hal/src/hal-sdcard.adb

```diff
@@ -0,0 +1,382 @@
+with Ada.Text_IO; use Ada.Text_IO;
```

Comment:
I can confirm this can't be moved to the services layer: the drivers layer that relies on this initialisation protocol don't see the services library (and if you add it to the device project, then you end up with a circular elaboration issue).